### PR TITLE
Update grammar for better mapping match

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Comment by temanbrcom** [comment]
@ap891843

**Review by asatklichov — APPROVED** [review]

File: server/src/test/java/com/broadcom/lsp/cobol/usecases/TestXmlQueryNotTerminatesSemanticAnalysis.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
           + "       ENVIRONMENT DIVISION.\n"
           + "       DATA DIVISION.\n"
           + "       LINKAGE SECTION.\n"
-          + "       01 {$*VARNAME} PIC 9.\n"
+          + "       01 VARNAME PIC 9.\n"
```
Comment (grianbrcom) [inline server/src/test/java/com/broadcom/lsp/cobol/usecases/TestXmlQueryNotTerminatesSemanticAnalysis.java:51]: Why are you exclude this variable definition?

Comment (temanbrcom) [reply server/src/test/java/com/broadcom/lsp/cobol/usecases/TestXmlQueryNotTerminatesSemanticAnalysis.java:51]: It is a leftover of the UseCaseEngine version of this test. At the moment, the preprocessing is not applied to this test.

**Comment by temanbrcom** [comment]
To be implemented as a part of #731

**Pull request closed without merging**